app/src/ui/changes/no-changes.tsx

```diff
@@ -697,8 +697,8 @@ export class NoChanges extends React.Component<
             <div className="text">
               <h1>No local changes</h1>
               <p>
-                You have no uncommitted changes in your repository! Here are
-                some friendly suggestions for what to do next.
+                There are no uncommitted changes to this repository. 
```

Comment:
Revisiting the whole message, I think this tweak might help overall:

```suggestion
                There are no uncommitted changes for this repository. 
```

Reply:
```suggestion
                There are no uncommitted changes for this repository on this machine. 
```
The ultimate combination of what we want?

Reply:
I'm not 100% sold on how the `on this machine` bit helps here. Let's talk about this a bit more in #8003.